src/Lifecycle.js

```diff
@@ -380,10 +380,21 @@ export function startMatrixClient() {
  */
 export function onLoggedOut() {
     _clearLocalStorage();
+    _clearIndexedDB();
     stopMatrixClient();
     dis.dispatch({action: 'on_logged_out'});
 }
 
+function _clearIndexedDB() {
+    // remove indexeddb instances
+    if (!window.indexedDB) {
+        return;
+    }
+    console.log("Clearing indexeddb");
+    window.indexedDB.deleteDatabase("matrix-js-sdk");
```

Comment:
Surely call `clearDatabase` on the indexdb object rather than relying on the hardcoded name of the store between 2 different codebases?